Docs pipeline failing

### What's failing?

Documentation build (Docusaurus)

### Branch

main

### GitHub Actions Run Link

https://github.com/coding-kitties/PyIndicators/actions/runs/22523562357

### Error Output

```shell
Run python scripts/charts/generate_all.py --download
Storage path /home/runner/work/PyIndicators/PyIndicators/scripts/charts/resources/data does not exist or is not a directory.
Storage path /home/runner/work/PyIndicators/PyIndicators/scripts/charts/resources/data does not exist or is not a directory.
Downloading market data...
Saved 1513 rows → OHLCV_BTC-EUR_BITVAVO_1d.csv
Saved 4692 rows → OHLCV_BTC-EUR_BITVAVO_4h.csv

Generating 52 chart(s)...

  [1/52] ERR  accumulation_distribution_zones: No module named 'scipy'
  [2/52] ERR  adx: No module named 'scipy'
  [3/52] ERR  atr: No module named 'scipy'
  [4/52] ERR  bearish_divergence: No module named 'scipy'
  [5/52] ERR  bollinger_bands: No module named 'scipy'
  [6/52] ERR  bollinger_overshoot: No module named 'scipy'
  [7/52] ERR  breaker_blocks: No module named 'scipy'
  [8/52] ERR  bullish_divergence: No module named 'scipy'
  [9/52] ERR  buyside_sellside_liquidity: No module named 'scipy'
  [10/52] ERR  crossover: No module named 'scipy'
  [11/52] ERR  crossunder: No module named 'scipy'
  [12/52] ERR  detect_peaks: No module named 'scipy'
  [13/52] ERR  ema: No module named 'scipy'
  [14/52] ERR  ema_trend_ribbon: No module named 'scipy'
  [15/52] ERR  fair_value_gap: No module named 'scipy'
  [16/52] ERR  fibonacci_retracement: No module named 'scipy'
  [17/52] ERR  golden_zone: No module named 'scipy'
  [18/52] ERR  golden_zone_signal: No module named 'scipy'
  [19/52] ERR  has_any_lower_then_threshold: No module named 'scipy'
  [20/52] ERR  internal_external_liquidity_zones: No module named 'scipy'
  [21/52] ERR  liquidity_levels_voids: No module named 'scipy'
  [22/52] ERR  liquidity_pools: No module named 'scipy'
  [23/52] ERR  liquidity_sweeps: No module named 'scipy'
  [24/52] ERR  macd: No module named 'scipy'
  [25/52] ERR  market_structure_break: No module named 'scipy'
  [26/52] ERR  market_structure_choch_bos: No module named 'scipy'
  [27/52] ERR  mitigation_blocks: No module named 'scipy'
  [28/52] ERR  momentum_confluence: No module named 'scipy'
  [29/52] ERR  moving_average_envelope: No module named 'scipy'
  [30/52] ERR  nadaraya_watson_envelope: No module named 'scipy'
  [31/52] ERR  opening_gap: No module named 'scipy'
  [32/52] ERR  optimal_trade_entry: No module named 'scipy'
  [33/52] ERR  order_blocks: No module named 'scipy'
  [34/52] ERR  premium_discount_zones: No module named 'scipy'
  [35/52] ERR  pulse_mean_accelerator: No module named 'scipy'
  [36/52] ERR  pure_price_action_liquidity_sweeps: No module named 'scipy'
  [37/52] ERR  rejection_blocks: No module named 'scipy'
  [38/52] ERR  rsi: No module named 'scipy'
  [39/52] ERR  sma: No module named 'scipy'
  [40/52] ERR  stochastic_oscillator: No module named 'scipy'
  [41/52] ERR  strong_weak_high_low: No module named 'scipy'
  [42/52] ERR  supertrend: No module named 'scipy'
  [43/52] ERR  supertrend_clustering: No module named 'scipy'
  [44/52] ERR  trendline_breakout_navigator: No module named 'scipy'
  [45/52] ERR  volume_imbalance: No module named 'scipy'
  [46/52] ERR  volume_weighted_trend: No module named 'scipy'
  [47/52] ERR  volumetric_supply_demand_zones: No module named 'scipy'
  [48/52] ERR  wilders_rsi: No module named 'scipy'
  [49/52] ERR  williams_r: No module named 'scipy'
  [50/52] ERR  wma: No module named 'scipy'
  [51/52] ERR  z_score_predictive_zones: No module named 'scipy'
  [52/52] ERR  zero_lag_ema_envelope: No module named 'scipy'

Done in 0.1s — 0 ok, 52 failed

Errors:
  accumulation_distribution_zones: No module named 'scipy'
  adx: No module named 'scipy'
  atr: No module named 'scipy'
  bearish_divergence: No module named 'scipy'
  bollinger_bands: No module named 'scipy'
  bollinger_overshoot: No module named 'scipy'
  breaker_blocks: No module named 'scipy'
  bullish_divergence: No module named 'scipy'
  buyside_sellside_liquidity: No module named 'scipy'
  crossover: No module named 'scipy'
  crossunder: No module named 'scipy'
  detect_peaks: No module named 'scipy'
  ema: No module named 'scipy'
  ema_trend_ribbon: No module named 'scipy'
  fair_value_gap: No module named 'scipy'
  fibonacci_retracement: No module named 'scipy'
  golden_zone: No module named 'scipy'
  golden_zone_signal: No module named 'scipy'
  has_any_lower_then_threshold: No module named 'scipy'
  internal_external_liquidity_zones: No module named 'scipy'
  liquidity_levels_voids: No module named 'scipy'
  liquidity_pools: No module named 'scipy'
  liquidity_sweeps: No module named 'scipy'
  macd: No module named 'scipy'
  market_structure_break: No module named 'scipy'
  market_structure_choch_bos: No module named 'scipy'
  mitigation_blocks: No module named 'scipy'
  momentum_confluence: No module named 'scipy'
  moving_average_envelope: No module named 'scipy'
  nadaraya_watson_envelope: No module named 'scipy'
  opening_gap: No module named 'scipy'
  optimal_trade_entry: No module named 'scipy'
  order_blocks: No module named 'scipy'
  premium_discount_zones: No module named 'scipy'
  pulse_mean_accelerator: No module named 'scipy'
  pure_price_action_liquidity_sweeps: No module named 'scipy'
  rejection_blocks: No module named 'scipy'
  rsi: No module named 'scipy'
  sma: No module named 'scipy'
  stochastic_oscillator: No module named 'scipy'
  strong_weak_high_low: No module named 'scipy'
  supertrend: No module named 'scipy'
  supertrend_clustering: No module named 'scipy'
  trendline_breakout_navigator: No module named 'scipy'
  volume_imbalance: No module named 'scipy'
  volume_weighted_trend: No module named 'scipy'
  volumetric_supply_demand_zones: No module named 'scipy'
  wilders_rsi: No module named 'scipy'
  williams_r: No module named 'scipy'
  wma: No module named 'scipy'
  z_score_predictive_zones: No module named 'scipy'
  zero_lag_ema_envelope: No module named 'scipy'
Error: Process completed with exit code 1.
```

### Steps to Reproduce

_No response_

### Recent Changes

_No response_

### Severity

🔴 Blocking — main branch is broken, no PRs can merge

### Deliverables

- [ ] Fix the failing test(s)
- [ ] Fix the build/lint issue
- [ ] Add missing test coverage to prevent regression
- [ ] Update CI workflow configuration

### Additional Context

_No response_

--- a/.github/workflows/deploy-docs.yml
+++ b/.github/workflows/deploy-docs.yml
@@ -37,6 +37,7 @@ jobs:
         run: |
           pip install --upgrade pip
           pip install poetry
+          poetry config virtualenvs.create false
           poetry install --no-interaction
           pip install plotly kaleido investing-algorithm-framework
 

